calculator.py

```diff
@@ -0,0 +1,39 @@
+import sys
+
+try:
+    symbol, x, y = sys.argv[1], sys.argv[2], sys.argv[3]
```

Comment:
It is better to use `num1` and `num2` instead of `x` and `y`. This will be more understandable

Reply:
Discussed